articles/how-to-manage-quantum-workspaces-with-the-azure-cli.md

```diff
@@ -60,8 +60,41 @@ And then you can create it using the following command, using the previous examp
 az quantum workspace create -l westus -g MyResourceGroup -w MyQuantumWorkspace -a MyStorageAccount
 ```
 
-> [!NOTE]
-> Adding or removing providers from a workspace using the Az CLI is not currenly supported. For now, you'll need to [use the Azure portal](xref:microsoft.quantum.workspaces-portal) to modify a workspace.
+By default, a new workspace will be created only containing the Microsoft Basic QIO provider. At any time, it is possible to modify a workspace in order to add or remove other providers. For this, you can use [the Azure portal](xref:microsoft.quantum.workspaces-portal) and edit the workspace.
+
+Alternatively, you can create a workspace in a more advanced way and specify the providers using the CLI directly.
+
+## Specify additional providers during creation of an Azure Quantum workspace
+
+1. To retrieve the list of quantum providers available, you can use the following command, using **westus** as example location:
+
+   ```dotnetcli
+   az quantum offerings list -l westus
+   ```
+
+1. You can see a summary table of the information above by specifying the output format:
+
+   ```dotnetcli
+   az quantum offerings list -l westus -o table
+   ```
+
+1. Once you determine the provider and SKU to include in your workspace, you can review terms using this command, assuming **MyProviderID** and **MySKU** as example values:
```

Comment:
What is an SKU?

Reply:
@geduardo , each provider offers multiple plans or SKUs (e.g. "Basic", "Premium", "Pay as you go") and each of them has associated terms/conditions, cost, etc.